libr/asm/p/asm_mips_gnu.c

```diff
@@ -17,6 +17,8 @@ static int mips_mode = 0;
 static unsigned long Offset = 0;
 static RStrBuf *buf_global = NULL;
 static unsigned char bytes[4];
+static char *pre_cpu = NULL;
```

Comment:
maybe good to start reducing the amount of globals in r2land

Reply:
i mean. im ok to do this now because binutils is full of global states, but we should be reducing the amount of globals

Reply:
got you